### ISSUE TITLE
Support Pinch Zoom on Desktop/Touchpad

### ISSUE THREAD

**Issue by simpros** [bluefireteam/photo_view#520]
It would be awesome to be able to zoom in by the default zoom gesture on a touchpad on desktop applications.

**Comment by renancaraujo** [comment]
Agreed, it shoudl be possible since https://github.com/flutter/flutter/issues/23604#issuecomment-1168106921 is merged
